package.json

```diff
@@ -2,15 +2,16 @@
   "name": "dash",
   "version": "3.14.0-dev.1",
   "description": "Dash library for JavaScript/TypeScript ecosystem (Wallet, DAPI, Primitives, BLS, ...)",
-  "main": "dist/dash.cjs.js",
+  "main": "build/src/index.js",
   "unpkg": "dist/dash.min.js",
   "types": "dist/src/index.d.ts",
   "scripts": {
     "start:dev": "nodemon --exec 'npm run build && npm run test:unit'",
-    "build": "webpack --display-error-details",
+    "build": "npm run compile && webpack --display-error-details",
     "check-package": "npm run check-package:name && npm run check-package:version",
     "check-package:name": "test $(jq -r .name package.json) = $(jq -r .name package-lock.json)",
     "check-package:version": "test $(jq -r .version package.json) = $(jq -r .version package-lock.json)",
+    "compile": "tsc",
```

Comment:
Let's call it `build:ts` and make a separate `build:browsers` (`npm run build:ts && webpack --display-error-details`) so we can keep `build` as browsers alias for now, but later probably it will be something else.

Reply:
I thought about it too, I couldn't make my mind, so I won't be against it. Especially as in the end, compilation (as per our config) will end up with typings in `dist/` and built es5 in `build/`. 
Will do : `build:ts`, `build:web` (you prefers it as `browsers` ?). 

Can you explain what will you see being in the `build` one ?

Reply:
Web is good! Build is just `npm run build:web` for now I suppose.